reportportal/v5/templates/migrations-job.yaml

```diff
@@ -9,6 +9,9 @@ spec:
       labels:
         component: {{ include "reportportal.fullname" . }}-migrations
     spec:
+      {{- if .Values.openshift.enable }}
+      serviceAccountName: {{ template "reportportal.serviceAccountName" . }}
```

Comment:
this service account has additional permissions to read some state over k8s api. Do we really this for openshift?